README.md

```diff
@@ -26,6 +26,11 @@ Please check out common DSC Resources [contributing guidelines](https://github.c
 * Initial release with the following resources 
     - xPhp
 
+### 1.0.?
```

Comment:
Hmmm... @vors seems like kind of a chicken/egg problem. Should we generate this from merged pull requests and add it to the changelog when we merge dev to master and rev the version?